CHANGELOG: update changelog for 1.6.0 release (#450)
linux-system-roles/network: CHANGELOG.md

Changelog
=========

[1.6.0] - 2022-02-02
--------------------

### New features

- Support routing tables in static routes

### Bug fixes

> - Fix setting DNS search settings only for enabled IP families

Reviewer: ```suggestion
- Fix setting DNS search settings if only one IP family is enabled
```

> - Fix reloading the configuration for nm provider on NetworkManager 1.18

Reviewer 2: I think the right description would be "Fix switching providers on NetworkManager 1.18". What do you think?

Reviewer: Technically, it is not only about switching the provider but also using the role the first time with the NM provider on a system that used network-scripts in the past and NM_CONTROLLED=no.

Reviewer: ```suggestion
- Fix switching from initscripts to NetworkManager 1.18
```

Author: I updated, thanks.
